Code/GraphMol/SmilesParse/nontetrahedral_tests.cpp

```diff
@@ -35,6 +35,7 @@ void inorganicSanitize(RDKit::RWMol &mol) {
 }
 
 TEST_CASE("bulk parse test") {
+  Chirality::setAllowNontetrahedralChirality(true);
```

Comment:
Duh, we really need a RAII class for this. I wonder if we should pull the ones I added in Code/GraphMol/catch_chirality.cpp to somewhere public.

Reply:
Yeah, I agree